docs/sources/developers/plugins/build-a-logs-data-source-plugin.md

```diff
@@ -4,19 +4,24 @@ title: Build a logs data source plugin
 
 # Build a logs data source plugin
 
-This guide explains how to build a logs data source plugin.
+Data logs are defined as data stored over a period of time in different systems or environments for the purpose of tracking certain events.  
```

Comment:
I've never come across the term "Data logs" and I'm not sure this definition is scoped enough to logs... I could say the same for metrics and alerts. Again I'm not sure we need to define logs

Reply:
Issue is fixed in latest commit.